### ISSUE TITLE
Are video tracks supported?

### ISSUE THREAD

**Issue by kimisgold** [sachinchoolur/lightGallery#1096]
I am looking to include subtitle tracks like the following markup:

```
<video id="video" controls preload="metadata">
   <source src="video/sintel-short.mp4" type="video/mp4">
   <source src="video/sintel-short.webm" type="video/webm">
   <track label="English" kind="subtitles" srclang="en" src="captions/vtt/sintel-en.vtt" default>
   <track label="Deutsch" kind="subtitles" srclang="de" src="captions/vtt/sintel-de.vtt">
   <track label="Español" kind="subtitles" srclang="es" src="captions/vtt/sintel-es.vtt">
</video>
```

This was fairly straightforward in v1 where I would provide the html5 video markup in a hidden div to be used in the `data-html` attribute. Is it possible to do this in v2?

**Comment by sachinchoolur** [comment]
Hey @kimisgold,

Just released version [2.2.0-beta.2](https://github.com/sachinchoolur/lightGallery/releases/tag/2.2.0-beta.2) with video tracks support.

Similar to the source tag, tracks can be passed via `data-video` attribute 

```html
data-video='{
  "source": [{"src":"/videos/video1.mp4", "type":"video/mp4"}], 
  "tracks": [{"src": "{/videos/title.txt", "kind":"captions", "srclang": "en", "label": "English", "default": "true"}], 
  "attributes": {"preload": false, "controls": true}
}'
```

```html
<a
        data-lg-size="1280-720"
        data-video='{"source": [{"src":"/videos/video1.mp4", "type":"video/mp4"}], "tracks": [{"src": "{/videos/title.txt", "kind":"captions", "srclang": "en", "label": "English", "default": "true"}], "attributes": {"preload": false, "controls": true}}'
        data-poster="/images/demo/youtube-video-poster.jpg"
        data-sub-html="<h4>'Peck Pocketed' by Kevin Herron | Disney Favorite</h4>"
    >
        <img
            width="300"
            height="100"
            class="img-responsive"
            src="/images/demo/youtube-video-poster.jpg"
        />
    </a>
```

I'll update the documentation by tomorrow. Let me know if you have any question

**Comment by kimisgold** [comment]
This is exactly what I'm looking for, thank you. I'll let you know if know if I run into any problems.
